Core/GDCore/IDE/Events/ExpressionValidator.cpp

```diff
@@ -88,12 +88,12 @@ ExpressionValidator::Type ExpressionValidator::ValidateFunction(const gd::Functi
   }
 
   if (!function.behaviorName.empty() &&
-      (globalObjectsContainer.HasObjectNamed(function.objectName) &&
-           !globalObjectsContainer.GetObject(function.objectName)
-                .HasBehaviorNamed(function.behaviorName) ||
-       objectsContainer.HasObjectNamed(function.objectName) &&
-           !objectsContainer.GetObject(function.objectName)
-                .HasBehaviorNamed(function.behaviorName))) {
+      ((globalObjectsContainer.HasObjectNamed(function.objectName) &&
+        !globalObjectsContainer.GetObject(function.objectName)
+             .HasBehaviorNamed(function.behaviorName)) ||
+       (objectsContainer.HasObjectNamed(function.objectName) &&
+        !objectsContainer.GetObject(function.objectName)
+             .HasBehaviorNamed(function.behaviorName)))) {
```

Comment:
This is unreleated. I the compiler for tests are more permissive so I didn't see the warnings about mixed operators.